packages/docs/stories/Welcome.mdx

```diff
@@ -0,0 +1,71 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Welcome" />
+
+<img src="/synergy_logo_transparent.png" height="100"></img>
+
+# Synergy Design System ![](/heart.svg)
+
+Welcome to the Synergy Design System: the centralized, living style guide for [SICK](https://www.sick.com) application user interfaces.
+
+Synergy gives you a head start for your application. Use colors, styles, components and assets to quickly spin-up your application in development ✍ and design 🎨 at the same time.
+Build multiple apps with different frameworks sharing the same UI components.
+
+## Sources &lt;/&gt;
+<div style={{display:"flex", alignItems:"center"}}><img src="/github.png" height="30"></img> &nbsp;The source code is available on&nbsp;[GitHub](https://github.com/synergy-design-system/synergy-design-system) </div><br/>
```

Comment:
Just a minor nitpick, but could we also add the links to the `<img />` tags themselfs?

Reply:
How would you change the SymbolLine component to achieve that?